vyxal/elements.py

```diff
@@ -415,13 +415,7 @@ def boolify(lhs, ctx):
     """Element ḃ
     (any) -> is truthy?
     """
-    if vy_type(lhs, simple=True) is list:
-        if ctx.truthy_lists:
-            return any_true(lhs, ctx)
-        else:
-            return vectorise(boolify, lhs, ctx=ctx)
-    else:
-        return int(bool(lhs))
+    return int(bool(lhs))
```

Comment:
If we are getting rid of the vectorised-ness, let's at least allow that old behavior under the `t` flag.

Reply:
Ok, working on it

Reply:
@cgccuser Done